types/wordpress__viewport/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Git is just wrong here -- we're deleting the first directory, and adding a completely different package.json in another directory :)